ShanghaiBloodSports/Packages/manifest.json

```diff
@@ -1,40 +1,41 @@
-{

```

Comment:
the way unity handles this diff hurts my soul

Reply:
:'(